package.json

```diff
@@ -65,6 +65,7 @@
     "ci:test": "npm run test",
     "clean": "rm -rf typechain-types cache bin dist artifacts contracts-exposed",
     "postclean": "hardhat clean",
+    "collector:change-partners": "hardhat change-partner-shares",
```

Comment:
Is the withdraw reference missing here in the list of scripts?

Reply:
yes, but we'll probably delete the change-partner script from here also, see #95

Reply:
done